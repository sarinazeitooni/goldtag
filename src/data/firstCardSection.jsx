let firstCard = [
    {
        gridArea: 'big1',
        id: 0,
        cover : "https://cdn.goldtag.net/web/uploads/pictures/736x536/999726840_1621418858__750-544.jpg",
        purchased : "44 عدد",
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1723945140_1621418866__750-5442.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/250610996_1621418862__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1443349497_1621418854__750-5443.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1723945140_1621418866__750-5442.jpg'
        ],
        title : "پالت سایه چشم HUDABEAUTY",
        subtitle : "18 رنگ، ماندگاری بالا",
        price : 90000,
        newprice : 64800,
        address : "پالت سایه چشم HUDABEAUTY < خانه > لوازم شخصی > لوازم آرایشی و بهداشتی",
        daynum : "16 روز",
        description : "پالت سایه چشم HUDA BEAUTY مدل New N.U.D.E دارای 18 رنگ رم و طبیعیه که این 18 رنگ شامل: 7 رنگ شاین و 10 رنگ مات و 1 رنگ کانسیلره، شما میتوانید با این پالت چشم انواع آرایش چشم از آرایش های ملایم و روزانه تا آرایش کامل شب رو داشته باشید. این پالت دارای ترکیب رنگی زیبا و منحصر به فردی بوده و از ماندگاری بالایی برخورداره.",
        property : [
            'پالت سایه',
            'طرح: HUDA BEAUTY',
            '18 رنگ',
            '7رنگ شاین و 10 رنگ مات و 1 رنگ کانسیلر',
            'با رنگهای گرم و طبیعی',
            'تنوع رنگی بالا',
            'قابلیت ایجاد انواع آرایش های چشم از آرایش ملایم و روزانه تا آرایش کامل شب',
            'دارای یک آینه بزرگ در قسمت درب پالت',
            'ابعاد: 17.5x10.5cm',
            'محصول: چین',
            'به دلیل رعایت مسائل بهداشتی امکان تعویض یا استرداد کالا وجود ندارد',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        tags : [
            'پالت سایه',
            'سایه چشم',
            'پالت سایه چشم HUDABEAUTY'    
        ],
        clothes : false
    }

];
export default firstCard;