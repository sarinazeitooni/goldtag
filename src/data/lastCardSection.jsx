let lastCard = [
    {
        gridArea: 'big2',
        id:9,
        title:"گلدان سرامیکی M.R.Z",
        subtitle: "دارای طرح گل رز",
        purchased: " 155 عدد",
        cover : "https://cdn.goldtag.net/web/uploads/pictures/270x170/704167149_1621340742__750-5444.jpg",
        price : 60000,
        newprice: 35400,
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/823778970_1621340749__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/157560287_1621340746__750-544.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/704167149_1621340742__750-5444.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/606712390_1621340757__750-5443.jpg'
        ],
        tags : [
            'گلدان',
            'گلدان سرامیکی',
            'گلدان سرامیکی M.R.Z'
        ],
        address: "خانه > لوازم خانگی > دکوری > گلدان سرامیکی M.R.Z",
        daynum: "15 روز",
        description: "امروز به شما گلدان سرامیکی رو پیشنهاد میکنیم که میتونید انواع گل طبیعی و مصنوعی رو داخلش بذارید. روی این گلدون طرح گل رز داره و دارای پکیجه و برای هدیه دادن میتونه انتخاب خوبی باشه.",
        property: [
            'گلدان',
            'جنس: سرامیک',
            'دارای طرح گل رز روی گلدان',
            'مناسب برای انواع گل طبیعی و مصنوعی',
            'مناسب برای هدیه',
            'ارتفاع گلدان: 24cm',
            'قطر دهانه گلدان: 3cm',
            'سطح مقطع: 5.5cm',
            'دارای پکیج', 
            'ابعادپکیج: 9.5x9.5x24cm',
            'محصول: چین',
            ' ارسال به شهرستان بدون امکان تعویض در صورت شکستگی',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        clothes : false
    },
    {
        gridArea: 'big3',
        id:10,
        cover : "https://cdn.goldtag.net/web/uploads/pictures/736x536/999726840_1621418858__750-544.jpg",
        purchased : "44 عدد",
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1723945140_1621418866__750-5442.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/250610996_1621418862__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1443349497_1621418854__750-5443.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1723945140_1621418866__750-5442.jpg'
        ],
        title : "پالت سایه چشم HUDABEAUTY",
        subtitle : "18 رنگ، ماندگاری بالا",
        price : 90000,
        newprice : 64800,
        address : "پالت سایه چشم HUDABEAUTY < خانه > لوازم شخصی > لوازم آرایشی و بهداشتی",
        daynum : "16 روز",
        description : "پالت سایه چشم HUDA BEAUTY مدل New N.U.D.E دارای 18 رنگ رم و طبیعیه که این 18 رنگ شامل: 7 رنگ شاین و 10 رنگ مات و 1 رنگ کانسیلره، شما میتوانید با این پالت چشم انواع آرایش چشم از آرایش های ملایم و روزانه تا آرایش کامل شب رو داشته باشید. این پالت دارای ترکیب رنگی زیبا و منحصر به فردی بوده و از ماندگاری بالایی برخورداره.",
        property : [
            'پالت سایه',
            'طرح: HUDA BEAUTY',
            '18 رنگ',
            '7رنگ شاین و 10 رنگ مات و 1 رنگ کانسیلر',
            'با رنگهای گرم و طبیعی',
            'تنوع رنگی بالا',
            'قابلیت ایجاد انواع آرایش های چشم از آرایش ملایم و روزانه تا آرایش کامل شب',
            'دارای یک آینه بزرگ در قسمت درب پالت',
            'ابعاد: 17.5x10.5cm',
            'محصول: چین',
            'به دلیل رعایت مسائل بهداشتی امکان تعویض یا استرداد کالا وجود ندارد',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        tags : [
            'پالت سایه',
            'سایه چشم',
            'پالت سایه چشم HUDABEAUTY'    
        ],
        clothes : false
    }


];
export default lastCard;