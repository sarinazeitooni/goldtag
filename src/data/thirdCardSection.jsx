let thirdCard = [
    {
        gridArea: 'small5',
        id:5,
        title:"گلدان سرامیکی M.R.Z",
        subtitle: "دارای طرح گل رز",
        purchased: " 155 عدد",
        cover : "https://cdn.goldtag.net/web/uploads/pictures/270x170/704167149_1621340742__750-5444.jpg",
        price : 60000,
        newprice: 35400,
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/823778970_1621340749__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/157560287_1621340746__750-544.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/704167149_1621340742__750-5444.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/606712390_1621340757__750-5443.jpg'
        ],
        tags : [
            'گلدان',
            'گلدان سرامیکی',
            'گلدان سرامیکی M.R.Z'
        ],
        address: "خانه > لوازم خانگی > دکوری > گلدان سرامیکی M.R.Z",
        daynum: "15 روز",
        description: "امروز به شما گلدان سرامیکی رو پیشنهاد میکنیم که میتونید انواع گل طبیعی و مصنوعی رو داخلش بذارید. روی این گلدون طرح گل رز داره و دارای پکیجه و برای هدیه دادن میتونه انتخاب خوبی باشه.",
        property: [
            'گلدان',
            'جنس: سرامیک',
            'دارای طرح گل رز روی گلدان',
            'مناسب برای انواع گل طبیعی و مصنوعی',
            'مناسب برای هدیه',
            'ارتفاع گلدان: 24cm',
            'قطر دهانه گلدان: 3cm',
            'سطح مقطع: 5.5cm',
            'دارای پکیج', 
            'ابعادپکیج: 9.5x9.5x24cm',
            'محصول: چین',
            ' ارسال به شهرستان بدون امکان تعویض در صورت شکستگی',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        clothes : false
    },
    {
        gridArea: 'small6',
        id: 6,
        cover : "https://cdn.goldtag.net/web/uploads/pictures/270x170/307803407_1621341999__750-5445.jpg",
        purchased : "231 عدد",
        title : "تیشرت مردانه طرح Nike",
        subtitle : "تک رنگ، 2 سایز",
        price : 50000,
        newprice : 35500,
        tags : [
            'تیشرت مردانه',
            'تیشرت مردانه طرح Nike',
            'تیشرت مردانه آستین کوتاه'  
        ],
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/83292830_1621342031__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/457949053_1618399929__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/307803407_1621341999__750-5445.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1285675403_1621342046__750-5444.jpg'
        ],
        address : "تیشرت مردانه طرح Nike < خانه > پوشاک > پوشاک مردانه",
        daynum : "15 روز",
        description : "تیشرت های مردانه طرح Nike در رنگ سفید ارائه شده و جنسش اسپان هستش، آستین کوتاه و یقه گرده. لوگوی چاپی روی تیشرت داره و دوختش تمیزه. 2 سایز داره که میتونید قبل از خرید به جدول سایزبندی مراجعه کنید تا انتخاب بهتری داشته باشید.",
        property : [
            'تیشزت مردانه',
            'جنس: اسپان',
            'طرح: Nike',
            'آستین کوتاه',
            'یقه گرد',
            'آستین کوتاه',
            'دارای لوگوی چاپی روی تیشرت',
            'رنگ سفید',
            'سایز: XL و 2XL',
            ' تخفیف این گلدتگ با توجه به سایز متفاوت است',
            'محصول: ایران',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        clothes : true
    },
    {
        gridArea: 'small7',
        id:7,
        cover : "https://cdn.goldtag.net/web/uploads/pictures/736x536/999726840_1621418858__750-544.jpg",
        purchased : "44 عدد",
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1723945140_1621418866__750-5442.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/250610996_1621418862__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1443349497_1621418854__750-5443.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/1723945140_1621418866__750-5442.jpg'
        ],
        title : "پالت سایه چشم HUDABEAUTY",
        subtitle : "18 رنگ، ماندگاری بالا",
        price : 90000,
        newprice : 64800,
        address : "پالت سایه چشم HUDABEAUTY < خانه > لوازم شخصی > لوازم آرایشی و بهداشتی",
        daynum : "16 روز",
        description : "پالت سایه چشم HUDA BEAUTY مدل New N.U.D.E دارای 18 رنگ رم و طبیعیه که این 18 رنگ شامل: 7 رنگ شاین و 10 رنگ مات و 1 رنگ کانسیلره، شما میتوانید با این پالت چشم انواع آرایش چشم از آرایش های ملایم و روزانه تا آرایش کامل شب رو داشته باشید. این پالت دارای ترکیب رنگی زیبا و منحصر به فردی بوده و از ماندگاری بالایی برخورداره.",
        property : [
            'پالت سایه',
            'طرح: HUDA BEAUTY',
            '18 رنگ',
            '7رنگ شاین و 10 رنگ مات و 1 رنگ کانسیلر',
            'با رنگهای گرم و طبیعی',
            'تنوع رنگی بالا',
            'قابلیت ایجاد انواع آرایش های چشم از آرایش ملایم و روزانه تا آرایش کامل شب',
            'دارای یک آینه بزرگ در قسمت درب پالت',
            'ابعاد: 17.5x10.5cm',
            'محصول: چین',
            'به دلیل رعایت مسائل بهداشتی امکان تعویض یا استرداد کالا وجود ندارد',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        tags : [
            'پالت سایه',
            'سایه چشم',
            'پالت سایه چشم HUDABEAUTY'    
        ],
        clothes : false
    },
    {
        gridArea: 'small8',
        id:8,
        title:"گلدان سرامیکی M.R.Z",
        subtitle: "دارای طرح گل رز",
        purchased: " 155 عدد",
        cover : "https://cdn.goldtag.net/web/uploads/pictures/270x170/704167149_1621340742__750-5444.jpg",
        price : 60000,
        newprice: 35400,
        swiper : [
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/823778970_1621340749__750-5441.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/157560287_1621340746__750-544.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/704167149_1621340742__750-5444.jpg',
            'https://cdn.goldtag.net/web/uploads/pictures/736x536/606712390_1621340757__750-5443.jpg'
        ],
        tags : [
            'گلدان',
            'گلدان سرامیکی',
            'گلدان سرامیکی M.R.Z'
        ],
        address: "خانه > لوازم خانگی > دکوری > گلدان سرامیکی M.R.Z",
        daynum: "15 روز",
        description: "امروز به شما گلدان سرامیکی رو پیشنهاد میکنیم که میتونید انواع گل طبیعی و مصنوعی رو داخلش بذارید. روی این گلدون طرح گل رز داره و دارای پکیجه و برای هدیه دادن میتونه انتخاب خوبی باشه.",
        property: [
            'گلدان',
            'جنس: سرامیک',
            'دارای طرح گل رز روی گلدان',
            'مناسب برای انواع گل طبیعی و مصنوعی',
            'مناسب برای هدیه',
            'ارتفاع گلدان: 24cm',
            'قطر دهانه گلدان: 3cm',
            'سطح مقطع: 5.5cm',
            'دارای پکیج', 
            'ابعادپکیج: 9.5x9.5x24cm',
            'محصول: چین',
            ' ارسال به شهرستان بدون امکان تعویض در صورت شکستگی',
            'عکس های اسلایدر، عکس واقعی از محصول هستند که توسط تیم گلدتگ گرفته شده اند و عکس اینترنتی نیستند'
        ],
        clothes : false
    }
]; export default thirdCard;