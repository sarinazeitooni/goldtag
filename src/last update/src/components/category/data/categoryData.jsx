const  categoryData = {
    h3text : 'گلدتگ براساس دسته ها',
    images : [
        'https://cdn.goldtag.net/web/uploads/banners/b1d1.jpg',
        'https://cdn.goldtag.net/web/uploads/banners/b2d1.jpg',
        'https://cdn.goldtag.net/web/uploads/banners/b5d1.jpg',
        'https://cdn.goldtag.net/web/uploads/banners/b4d1.jpg',
        'https://cdn.goldtag.net/web/uploads/banners/b3d1.jpg'
    ]
}
export default categoryData;