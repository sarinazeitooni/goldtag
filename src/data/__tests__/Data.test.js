import React from 'react';
import { shallow } from 'enzyme';
import data from "../Data";
describe("data text" , ()=>{
    it("it should render",()=>{
        const container = shallow(<data/>)
        expect(container).toMatchSnapshot();
    });
});