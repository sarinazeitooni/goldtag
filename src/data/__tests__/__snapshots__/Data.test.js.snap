// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data text it should render 1`] = `ShallowWrapper {}`;
